pkg/operator/podsecurityreadinesscontroller/violation.go

```diff
@@ -0,0 +1,72 @@
+package podsecurityreadinesscontroller
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	applyconfiguration "k8s.io/client-go/applyconfigurations/core/v1"
+	"k8s.io/klog/v2"
+	psapi "k8s.io/pod-security-admission/api"
+)
+
+var podSecurityAlertLabels = []string{
+	psapi.AuditLevelLabel,
+	psapi.WarnLevelLabel,
+}
+
+func (c *PodSecurityReadinessController) isNamespaceViolating(ctx context.Context, ns *corev1.Namespace) (bool, error) {
```

Comment:
Similarly as previous comment, `is` prefix not very common in go; how about `namespaceViolating()`?

Reply:
From the standard library: [IsUpper](https://pkg.go.dev/unicode#IsUpper)